fl4health/server/scaffold_server.py

```diff
@@ -37,22 +53,30 @@ def __init__(
         )
         self.warm_start = warm_start
 
-    def initialize_paramameters(self, timeout: Optional[float]) -> None:
+    def _get_initial_parameters(self, timeout: Optional[float]) -> Parameters:
```

Comment:
Rather than having to worry about `_get_initial_parameters` being called after the custom `initialize_paramameters` function for warm-start, we just override the parent `_get_initial_parameters` so that we have control over when the warm start work happens.